bindings/sundials4py/test/test_sunadjointstepper.py

```diff
@@ -0,0 +1,92 @@
+# -----------------------------------------------------------------
+# Programmer(s): Cody J. Balos @ LLNL
+# -----------------------------------------------------------------
+# SUNDIALS Copyright Start
+# Copyright (c) 2025-2025, Lawrence Livermore National Security,
+# University of Maryland Baltimore County, and the SUNDIALS contributors.
+# Copyright (c) 2013-2025, Lawrence Livermore National Security
+# and Southern Methodist University.
+# Copyright (c) 2002-2013, Lawrence Livermore National Security.
+# All rights reserved.
+#
+# See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# SUNDIALS Copyright End
+# -----------------------------------------------------------------
+# Unit/smoke tests for SUNAdjointStepper module
+# -----------------------------------------------------------------
+
+import pytest
+from fixtures import *
+from sundials4py.core import *
+
+
+def make_adjoint_stepper(sunctx, sunstepper, nvec):
+    mem_helper = SUNMemoryHelper_Sys(sunctx)
+    status, scheme = SUNAdjointCheckpointScheme_Create_Fixed(0, mem_helper, 1, 1, 0, sunctx)
+    b1 = 0
+    b2 = 0
```

Comment:
Do `True` and `False` get converted as expected?
```suggestion
    own_stepper = False
```